startup.d/10-git-pull.sh

```diff
@@ -1,2 +1,23 @@
 #!/bin/sh
+
+# Prepare code auditing environment
+REPO_URL=$(git remote get-url origin)
+TEST_DIR=$(mktemp -d)
+git clone "$REPO_URL" "$TEST_DIR";
+pushd "$TEST_DIR";
+
+# Code auditing section
+find . -name '*.py' -print0 | xargs -0 python -m py_compile;
```

Comment:
Looks like I commented on the wrong PR.

`python3 -m py_compile **/*.py` would be cleaner.